sdc/tests/test_dataframe.py

```diff
@@ -1473,30 +1482,67 @@ def test_impl(df, df2):
         df = pd.DataFrame({'A': np.arange(n), 'B': np.arange(n)**2})
         df2 = pd.DataFrame({'A': np.arange(n), 'C': np.arange(n)**2})
         df2.A[n // 2:] = n
+        hpat_func(df, df2)
         pd.testing.assert_frame_equal(hpat_func(df, df2), test_impl(df, df2))
 
-    @unittest.skip("needs dict typing in Numba")
+    @skip_sdc_jit
+    def test_isin_df2(self):
```

Comment:
Please make all tests have descriptive names, so that it's clear what the test actually verifies, e.g. if the purpose of this one is to verify isin works with dfs with indexes, you could name it test_isin_df_index_str.